earth_enterprise/src/support/geecheck/geecheck.pl

```diff
@@ -125,7 +125,8 @@ BEGIN
                       "5.1.3" => ["1", 0],
                       "5.2.0" => ["1", 0],
                       "5.2.1" => ["1", 0],
-                      "5.2.2" => ["1", 0],                      
+                      "5.2.2" => ["1", 0],
+                      "5.2.3" => ["1", 0],                      
```

Comment:
I won't hold up merging because of this, but I'll point out that we are carrying 22 spaces at the end of the final line of this array. I say carrying because the spaces were present before these changes.

This is a really minor thing, but causes the diffs to be a smidge noisier.